test/testdata/bootstrappolicy/bootstrap_cluster_role_bindings.yaml

```diff
@@ -821,6 +821,36 @@ items:
   - kind: ServiceAccount
     name: template-instance-controller
     namespace: openshift-infra
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    creationTimestamp: null
+    name: system:openshift:controller:template-instance-finalizer-controller
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: system:openshift:controller:template-instance-finalizer-controller
+  subjects:
+  - kind: ServiceAccount
+    name: template-instance-finalizer-controller
+    namespace: openshift-infra
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    creationTimestamp: null
+    name: admin
```

Comment:
ah, I see.  Makes sense now.

Reply:
> ah, I see. Makes sense now.

Name doesn't look good through.  `system:openshift:controller:<something>`

Reply:
i used the same name that templateinstancecontroller was using...should that be changed too?  (will reconciliation handle it?)

Reply:
(actually this is what `NewClusterBinding` gives back by default as the name..uses the rolename)